defaults/main.yml

```diff
@@ -7,3 +7,6 @@ security_sudoers_passwordless: []
 security_sudoers_passworded: []
 
 security_autoupdate_enabled: true
+
+security_autoupdate_mail_to: null
```

Comment:
I'd rather just set this to an empty string—less duck typing results in less weird behavior down the road. We can test for an empty string and having `""` makes it more obvious the user should enter an email address as a string.